cli/status/status.go

```diff
@@ -20,35 +68,110 @@ type StatusOpts struct {
 func Status(runningCtx running.RunningCtx, opts StatusOpts) error {
 	ts := table.NewWriter()
 	ts.SetOutputMirror(os.Stdout)
-	ts.AppendHeader(table.Row{"INSTANCE", "STATUS", "PID", "MODE"})
+	ts.AppendHeader(table.Row{"INSTANCE", "STATUS", "PID", "MODE", "ALERTS"})
 
 	for _, run := range runningCtx.Instances {
 		row := []interface{}{}
 		fullInstanceName := running.GetAppInstanceName(run)
 		procStatus := running.Status(&run)
-
 		row = append(row, fullInstanceName)
-		row = append(row, procStatus.ColorSprint(procStatus.Status))
-		if procStatus.Code == process_utils.ProcessRunningCode {
-			row = append(row, procStatus.PID)
-		}
+		warnings := []string{}
+		mode_str := ""
 
 		conn, err := connector.Connect(connector.ConnectOpts{
 			Network: "unix",
 			Address: run.ConsoleSocket,
 		})
 		if err == nil {
-			res, err := conn.Eval("return (type(box.cfg) == 'function') or box.info.ro",
+			res, err := conn.Eval("return require('config'):info()",
+				[]any{}, connector.RequestOpts{})
+
+			if err == nil && len(res) != 0 {
+				configInfo := res[0].(map[string]interface{})
+
+				status := configInfo["status"]
+				if status == "startup_in_progress" {
+					warnings = append(warnings,
+						fmt.Sprintf("[Status]: %s", status))
+				}
```

Comment:
I would notify a user about any status except `ready`.

Reply:
There can be several statuses here (from `box.info.status` and `config.info.status`), I decided to combine them and write them together in case `status != ready`. For example `check_errors/orphan`. If everything is ok - then `status = ready`. I left some screenshots below.